packages/mds-audit/attachments.ts

```diff
@@ -0,0 +1,174 @@
+/*
+    Copyright 2019 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+import db from '@mds-core/mds-db'
+import log from '@mds-core/mds-logger'
+import aws from 'aws-sdk'
+import path from 'path'
+import sharp from 'sharp'
+import uuid from 'uuid'
+import { Attachment, AttachmentSummary, AuditAttachment, Recorded, UUID } from '@mds-core/mds-types'
+import { UnsupportedTypeError, ValidationError } from '@mds-core/mds-utils'
+/* eslint-disable-next-line */
+const multer = require('multer')
+
+/* eslint-disable-next-line */
+aws.config.getCredentials(async (err) => {
+  if (err) {
+    await log.error('Error getting AWS credentials', err.stack || err)
+  } else if (aws.config.credentials) {
+    aws.config.update({
+      secretAccessKey: aws.config.credentials.secretAccessKey,
+      accessKeyId: aws.config.credentials.accessKeyId,
+      region: aws.config.region
+    })
+  }
+})
+const supportedMimetypes = ['image/png', 'image/jpeg']
+const thumbnailSize = 250
+const s3Bucket = 'lacuna-audit-uploads'
+const s3BucketSubdir = 'dev.mdscompliance.app'
```

Comment:
Most of these s3 constants should be environment variables

Reply:
Updated. We'll need to get them into the build scripts cc @invertigo